mirage/views/node.ts

```diff
@@ -0,0 +1,14 @@
+import { HandlerContext, Schema } from 'ember-cli-mirage';
+
+export function createNode(this: HandlerContext, schema: Schema) {
+    const attrs = this.normalizedRequestAttrs();
+    const node = schema.nodes.create(attrs);
+    const now = (new Date()).toISOString();
+    node.attrs.dateModified = now;
+    node.attrs.dateCreated = now;
```

Comment:
Minor: I think it'd be more idiomatic to pass `dateModified`/`dateCreated` to `create()`
```ts
    const attrs = this.normalizedRequestAttrs();
    const now = (new Date()).toISOString();
    const node = schema.nodes.create({
        ...attrs,
        dateModified: now,
        dateCreated: now,
    });
```